app/views/development_metrics/defect_escape_rate/_values_details.erb

```diff
@@ -18,7 +18,11 @@
         <h5>Bugs by environment:</h5>
         <% metric[:values].each do |metric| %>
           <div class="item-list">
+            <% if metric.first -%>
             <%= metric.first.to_s.capitalize %>:
+            <% else -%>
+              None:
```

Comment:
we are missing a test checking for `None:`

Reply:
Done!